pages/apps/maps/index.vue

```diff
@@ -127,13 +127,15 @@ const checkSpecies = (route, organ, organ_name, taxo, for_species) => {
       route.query.for_species !== flatmaps.speciesMap[taxo]
     ) {
       failMessage = `Sorry! A flatmap for ${for_species} species does not yet exist. The ${organ_name} of a rat has been shown instead.`
+    } else if (!organ) {
+      failMessage = `Sorry! A entity on the anatomical structure does not yet exist. A generic flatmap for ${for_species} species has been shown instead.`
```

Comment:
Probably change this one to `Sorry! Applicable entity is not yet available. A generic flatmap for ${for_species} species has been shown instead.`